### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by licaon-kter** [nikita36078/J2ME-Loader#348]
The readme states: `This project is a fork of J2meLoader` and `Copyright 2017-2018 Nikita Shakarun. Licensed under the Apache License, Version 2.0.`.

But looking at the original repo ( https://github.com/NaikSoftware/J2meLoader ) I can't find an actual LICENSE file or info on what that code was licensed under.

How did this become *Licensed under the Apache License, Version 2.0* ?

Thanks

**Comment by nikita36078** [comment]
Yes, the original code wasn't licensed, so I asked author (Nickolay Savchenko aka NaikSoftware) for permission to license his code and he agreed.
